src/launch/lib/service/template/launchconfig.py

```diff
@@ -34,26 +34,28 @@ def properties_file(self, value: dict, current_path: Path, dest_base: Path) -> N
                 pass
 
     def copy_additional_files(
-        self, value: dict, current_path: Path, repo_base: Path, dest_base: Path
+        self, value: dict, current_path: Path, dest_base: Path
     ) -> None:
-        # print(f"\n\n{value=}\n{current_path=}\n{repo_base=}\n{dest_base=}\n\n")
-        # breakpoint()
         for target_file, source_file in value[
             LAUNCHCONFIG_KEYS.ADDITIONAL_FILES.value
         ].items():
-            source_path = repo_base.joinpath(source_file)
-            target_path = Path(current_path).joinpath(target_file)
+            file_path = Path(source_file).resolve()
+            target_path = current_path.joinpath(target_file)
+            value[LAUNCHCONFIG_KEYS.ADDITIONAL_FILES.value][target_file] = str(
+                f"./{target_path.relative_to(dest_base)}"
+            )
+            
             if self.dry_run:
                 click.secho(
-                    f"[DRYRUN] Copying additional file, would have copied {source_path} to {target_path}",
+                    f"[DRYRUN] Processing template, would have copied: {file_path} to {target_path}",
                     fg="yellow",
                 )
             else:
-                target_path.parent.mkdir(parents=True, exist_ok=True)
-                shutil.copy(src=source_path, dst=target_path)
-                click.echo(
-                    f"Copied additional file from {source_path} to {target_path}"
-                )
+                try:
+                    target_path.parent.mkdir(parents=True, exist_ok=True)
+                    shutil.copy(file_path, target_path)
+                except shutil.SameFileError:
+                    pass
```

Comment:
This is acceptable as-is, but I wanted to provide you with an alternative for the future that I find to be a lot more readable:

```python
from contextlib import suppress

with suppress(shutil.SameFileError):
    # your code that might raise a SameFileError that you wish to ignore
```